ReactNativeClient/PluginAssetsLoader.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+var __awaiter = (this && this.__awaiter) || function(thisArg, _arguments, P, generator) {
+	function adopt(value) { return value instanceof P ? value : new P(function(resolve) { resolve(value); }); }
+	return new (P || (P = Promise))(function(resolve, reject) {
+		function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
+		function rejected(value) { try { step(generator['throw'](value)); } catch (e) { reject(e); } }
+		function step(result) { result.done ? resolve(result.value) : adopt(result.value).then(fulfilled, rejected); }
+		step((generator = generator.apply(thisArg, _arguments || [])).next());
+	});
+};
+Object.defineProperty(exports, '__esModule', { value: true });
+const { dirname } = require('lib/path-utils.js');
+const { shim } = require('lib/shim');
+const Setting = require('lib/models/Setting');
+const pluginAssets = require('./pluginAssets/index');
+const KvStore = require('lib/services/KvStore.js');
+class PluginAssetsLoader {
```

Comment:
This file shouldn't be committed (I think if you rebase, it's now included in .gitignore).